terra/templates/terra/unit.html

```diff
@@ -16,12 +16,23 @@
 {% block body %}
     <div class="row">
         <div class="col">
-            <h1>Fiscal Year {{fiscalyear}}</h1>
+             <h1>Fiscal Year {{fiscalyear}}</h1> 
             <h2>{{unit.name}}</h2>
             <h3>{{unit.manager}}</h3>
         </div>
         <div class="col text-right">
-            <a href="/unit/{{unit.pk}}/export/" class="btn btn-primary" role="button" aria-pressed="true">Download CSV</a>
+            <a href="{% url 'unit_csv' pk=unit.id year=fy %}" class="btn btn-primary" role="button" aria-pressed="true">Download CSV</a>
+        </div>
+
+        <div class="dropdown">
+            <button class="btn btn-light dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Fiscal Year
+            </button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <a class="dropdown-item" href="{% url 'unit_detail' pk=unit.id year=2021 %}">FY2021</a>
+                <a class="dropdown-item" href="{% url 'unit_detail' pk=unit.id year=2020 %}">FY2020</a>
```

Comment:
This should be generated rather than hard coded. We could use a setting for inception date of the system and assume that now data older than that fiscal year exists. Then you can generate the dropdown options from current FY+1 backwards to inception date.